test/check-docker

```diff
@@ -50,12 +50,7 @@ class TestDocker(MachineCase):
 
     def wait_for_message(self, message):
         b = self.browser
-        # HACK - sometimes container output is missing from the logs
-        #        https://github.com/docker/docker/issues/10617
```

Comment:
Let's leave a comment here with a link to the bug as a clue why this might be failing.